### ISSUE TITLE
Failing offline-snapshot due to configured Scaling Policy

### ISSUE THREAD

**Issue by mbloch1986** [shinesolutions/aem-aws-stack-builder#295]
In some very rare cases the offline-snapshot may fail at the beginning of the process due to the configured ASG scaling policy for the publish-dispatcher ASG.

**How to reproduce:**

It's very difficult to reproduce as this scenario is very rare. But if this happens you'll see in  the Publish-Dispatcher ASG that the minimum instance count was updated but no instances were set in Standby. In the `Activity History` of the ASG you will see a Termination event which was caused by the Down Scale scaling policy.

**Problem description:**

The Autoscaling Group for Publish-Dispatcher has per default a Upscale & Downscale scaling policy configured. When the offline-snapshot process starts one of the very first step is to update the minimnum instance count for the Publish-Dispatcher ASG(code: https://github.com/shinesolutions/aem-stack-manager-cloud/blob/master/lambda/aem_offline_snapshot.py#L195). Afterwards the process is moving one instance of the Publish-Dispatcher into standby (https://github.com/shinesolutions/aem-stack-manager-cloud/blob/master/lambda/aem_offline_snapshot.py#L202). 

This whole process is happening in less than a second but in some cases between updating the ASG & moving one instance into standby the ASG down scale policy will terminate one instance, as it's scaling down. 

To avoid any scaling events during the offline-snapshot we need to update the process to also update the `suspended processes` to suspend any EC2 instance terminations.

**Suggested solution**:

Before updating the Auto Scaling group & moving instances to standby and from standby back to active, we should update the suspendes processes and suspend `termination` and resume after the offfline-snapshot is done.

Suspend processes:
```
autoscaling.suspend_processes(
    AutoScalingGroupName=asg_name,
    ScalingProcesses=[
        'Terminate',
    ]
)
```

Resume processes:
```
autoscaling.resume_processes(
    AutoScalingGroupName=asg_name,
    ScalingProcesses=[
        'Terminate',
    ]
)
```

**Comment by mbloch1986** [comment]
Instead of suspending/resuming the `Terminate` process we should rather suspend/resume the `AlarmNotification` process, since the AlarmNotification is activating the scaling policy.

**Comment by mbloch1986** [comment]
Fix is included in release 4.19.0
